tensorflow/core/distributed_runtime/worker_cache_logger.h

```diff
@@ -58,7 +58,9 @@ class WorkerCacheLogger {
   // later retrieval by RetrieveLogs().
   void RecordRecvTensor(int64 step_id, int64 start_usecs, int64 end_usecs,
                         const string& tensor_name, const string& src_device,
-                        const string& dst_device, int64 bytes);
+                        const string& dst_device, int64 bytes,
+                        const string& node="",
+                        const string& title="RecvTensor");
```

Comment:
Instead of adding default arguments to RecordRecvTensor, can you keep RecordRecvTensor as-is, and add a new method called "RecordDataTransfer()" that takes the additional (mandatory) arguments?

(Two motivations here: (i) we don't use default args very much in the codebase, and (ii) if you can change "RecvTensor" to something else, it's not really recording a "RecvTensor".)